app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
What just came to my mind: We shouldn't trust `receiving` when importing (so that is something we need to remember when adding the import functionality), because otherwise it is a privacy issue, when somebody adds contacts with `receiving` true to their export. But we still need it in the export (so the new format is OK), because we need to send the `account_migration` message to all contacts, even when only `receiving` is true.

For the import, the contact should resend a `contact` message when receiving the `account_migration` message, so we can recreate the contact with `receiving = true`. That's something that can already be added to #6750 at the end of `AccountMigration.perform!`: If new person is remote, then send new `contact` for all users who are sharing with the old person.